test/e2e/gateway/api_grpc_test.go

```diff
@@ -1058,14 +1054,19 @@ func TestGetUserAttributeKeys(t *testing.T) {
 	}
 
 	foundAttributes := make(map[string]bool)
```

Comment:
If the boolean is not used (It's always set to true), the `map[string]struct{}` is lighter and better.
For testing is fine, but keep this in mind when developing for production.